source/common/network/BUILD

```diff
@@ -250,25 +250,43 @@ envoy_cc_library(
         "io_socket_handle_impl.cc",
         "socket_interface_impl.cc",
         "win32_socket_handle_impl.cc",
-    ],
+    ] + select({
+        "//bazel:linux": [
+            "io_uring_socket_handle_impl.cc",
+        ],
```

Comment:
Does this mean we still build those iouring code for mobile? I'm curious that why mobile coverage test is ok for this.

Reply:
It look like `//bazel:linux` only builds in the real Linux (else it may fail the CI).

Reply:
Yea, the CI also passes after removing the file.